docs/faq.md

```diff
@@ -114,3 +114,18 @@ For the Example project, run the `EarlGreyExampleSwiftTests` target once then fi
 For physical device builds, replace `Debug-iphonesimulator` with `Debug-iphoneos`.
 
 You can verify the `EarlGrey.framework` is included in the bundle at `EarlGreyExampleSwiftTests.xctest/Frameworks/EarlGrey.framework`
+
+**How should I handle animations?**
+
+Disabling animations improves test performance and eliminates a source of flakiness.
+
+```swift
+UIView.AnimationsEnabled = false
+```
+
+Alternatively, the animation speed can be increased to improve coverage of the app.
+This enables finding bugs that otherwise wouldn't be triggered if we simply disabled animations.
+
+```swift
+UIApplication.sharedApplication().keyWindow!.layer.speed = 100
```

Comment:
The force unwrapping is not needed so it can be written in this style:
`UIApplication.sharedApplication().keyWindow?.layer.speed = 100`